src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js

```diff
@@ -78,32 +78,37 @@ kunstmaanbundles.pagepartEditor = (function(window) {
                     elem = $select.closest('.js-sortable-item').after(data);
                 }
 
+                // Create a temporary node of the new PP
+                var $temp = $('<div>');
+                $temp.append(data);
+
+                // Check if some javascript needs to be reinitialised for this PP
+                $temp.find('.js-reinit').each(function() {
```

Comment:
You can just search for elements with the reinit-js data attribute, so we don't really need this class. That's one less thing you can forget to make this work :)

````
$temp.find('*[data-reinit-js]')...
````